src/cluster.c

```diff
@@ -258,7 +258,7 @@ void restoreCommand(client *c) {
             rewriteClientCommandArgument(c,c->argc,shared.absttl);
         }
     }
-    objectSetLRUOrLFU(obj,lfu_freq,lru_idle,lru_clock,1000);
+    objectSetLRUOrLFU(kv, lfu_freq, lru_idle, lru_clock, 1000);
```

Comment:
same here.

Reply:
This function is good example of where we better strive to a clear distinction between plain object and kvobj.